### DISCUSSION TITLE
Infolist inside Form wizard

### DISCUSSION THREAD

**Discussion by ALSHIHAB** [filamentphp/filament#8529] [Help]
Is it possible to call Infolist from wizard called from getSteps()? Actually I need it to review steps before submission.

**Comment by leandrocfe** [comment]
You can create a [custom livewire component](https://filamentphp.com/docs/3.x/infolists/adding-an-infolist-to-a-livewire-component#setting-up-the-livewire-component) to achieve this. But I think you could use [placeholder](https://filamentphp.com/docs/3.x/forms/layout/placeholder) to do this

**Reply by ALSHIHAB** [reply]
I need the oposite. livewire component inside wizard step. I think we can use View::make([livewire component]). But how to properly call this and insure it pass the data as well?

**Reply by dave-mills** [reply]
@ALSHIHAB - I've been exploring the same thing. I think @leandrocfe might be suggesting making the placeholders dynamic, as  described [here](https://filamentphp.com/docs/3.x/forms/layout/placeholder#dynamically-generating-placeholder-content). For example:

```php 

public function getSteps(): array
{
    Step::make('Step 1')
        ->schema([
            // **** some example fields ****
            TextInput::make('title'),
            TextArea::make('description'),
]),
    Step::make('Step 2')
        ->schema([ 
            // **** some more example fields ****
            TextInput::make('another_field'),
]),
    Step::make('Review Data')
        ->schema([
            // **** For each field, make a placeholder. The contents are dynamic using the $get() utility injection ****
            Placeholder::make('review_title')
                 ->content(fn(Get $get): string => $get('title') ?? ''),
            Placeholder::make('review_description')
                 ->content(fn(Get $get): string => $get('description') ?? ''),
            Placeholder::make('review_another_field')
                 ->content(fn(Get $get): string => $get('another_field') ?? ''),
]),
```

That might be enough? If you need more control over how the review step looks, you could explore making your own custom field, that can receive data in the same way (using $get). Then you can render it however you want by customising the blade template. I don't know if you can embed an actual InfoList component inside a custom Field, but maybe that's worth looking into?
